README.md

```diff
@@ -1,6 +1,11 @@
-[![Build Status](https://travis-ci.org/CGAL/cgal.svg?branch=master)](https://travis-ci.org/CGAL/cgal)
+<p align="center"><img src="Installation/doc_html/images/cgal_2013_grey.png" width="400"></p>
+
+<p align="center">
+<a href="https://travis-ci.org/CGAL/cgal"><img src="https://travis-ci.org/CGAL/cgal.svg?branch=master"></a></a>
```

Comment:
- order is probably better
- Why twice `</a></a>` ? Probably intended `</img></p>`
- build status, I don't think this should be centered, but maybe a text in front of it plus the inline image.

Reply:
Actually, a lot of big repositories like laravel have their readme in this format i.e logo centered and build status below the logo. That is why i felt like it would look good if we do the same here as well.